sdk/core/azure-core-opentelemetry/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# Release History
```

Comment:
Question on naming and folder structure: How do other languages name this (and the namespace)?

For example, I am seeing tracing in there as well, like core-tracing-opentelemetry:
https://docs.microsoft.com/en-us/python/api/overview/azure/core-tracing-opentelemetry-readme?view=azure-python-preview

Reply:
For .Net, opentelemetry is built into the runtime.
For Java, it is azure-core-tracing-opentelemetry,
For JS/TS, it is @Azure/core-tracing
For Python, it is azure-core-tracing-opentelemetry

So it makes sense to rename the directory to azure-core-tracing-opentelemetry